pyret/visualizations.py

```diff
@@ -283,6 +285,9 @@ def spatial(filt, maxval=None, **kwargs):
         The filter whose spatial component is to be plotted. It may have
         temporal components.
 
+    dx : float, optional
+        The spatial size of one pixel, setting the scale of the x- and y-axes.
```

Comment:
s/"spatial size of one pixel"/"spatial sampling rate (microns/pixel)"

Reply:
"Spatial sampling rate" is good, but I think we should remain ambiguous about the actual units. Users may want to plot in microns, millimeters, or whatever.

Reply:
Sorry, "agnostic", not "ambiguous".